### ISSUE TITLE
Please show 'Steam Play Compatibility: Completely|Stable Stable|Unstable|Unplayable|Crashes|Won't Start" to Store

### ISSUE THREAD

**Issue by kisak-valve** [ValveSoftware/steam-for-linux#5808]
Issue transferred from https://github.com/ValveSoftware/Proton/issues/1672.
@ShannonVanWagner posted on 2018-10-02T03:57:38:

# Feature Request

## I confirm:
- [X] that I haven't found another request for this feature.
- [X] that I have checked whether there are updates for my system available that
      contain this feature already.

## Description
Please show 'Steam Play Compatibility: Completely|Stable Stable|Unstable|Unplayable|Crashes|Won't Start" + a link to the compatibility page for the game to Store pages for all games in the steam interface. This would enable me to quickly decide if I want to purchase the game or not, based on compatibility with my Linux via Steam Play.

## Justification [optional]
When I search through the steam store, I'm looking for games that work with my Linux OS.
***
@lucifertdark commented on 2018-10-02T07:11:39

There's already something similar to what you want
https://addons.mozilla.org/en-CA/firefox/addon/spcr-for-steam/
it adds onto the steam page the spcr rating for quite a few games.
***
@tr37ion commented on 2018-10-02T10:40:06

The intention is good. From a marketing perspective it is not so good idea to show mostly negative information of games not running on Steam. It might be something to add to dev's backend interface.

PS: Please don't use bold fonts often. This might be counter-productive to your intention. Keep your writing simple and lean on Github.
***
@HonkingGoose commented on 2018-10-02T10:44:45

Would adding a rating from SPCR really be helpful though? I feel that a rating on its own is no use, because you need to look at the reports themselves, to see if your similar configuration will work. 

A Platinum rating for example might only be relevant for users on Ubuntu 18.04, with Nvidia cards, with the proprietary 396.54 drivers. If you're using Fedora with an AMD gpu with MESA drivers things might not work. So a Fedora user sees the Platinum rating for Ubuntu and thinks, oh it works, buys game, and then finds out that the game doesn't work for them.
So any kind of rating system, I feel, should only show ratings for as close to your system as possible, to be of any use...

So blanket ratings are not really helpful after all, because those ratings are generally only relevant for precisely that configuration reported.

@ShannonVanWagner I think you made a mistake with filling in the pre-filled report. You have made one big header instead of header and then text.

I think if you edit your report to show this, it will show up like you wanted it to show up:

>\\# Description
Please show 'Steam Play Compatibility: Completely|Stable Stable|Unstable|Unplayable|Crashes|Won't Start" + a link to the compatibility page for the game to Store pages for all games in the steam interface. This would enable me to quickly decide if I want to purchase the game or not, based on compatibility with my Linux via Steam Play.
>\\# Justification
When I search through the steam store, I'm looking for games that work with my Linux OS.

Basically after a # sign, you should type the text for the header, and then press ENTER for a newline.

There is also a very handy [tutorial](https://commonmark.org/help/tutorial/)  for common Markdown, only takes 10 minutes and you're up to speed for the most part. Also try looking at: https://help.github.com/articles/basic-writing-and-formatting-syntax/

**Comment by Xavron** [comment]
> The intention is good. From a marketing perspective it is not so good idea to show mostly negative information of games not running on Steam. It might be something to add to dev's backend interface.

Its also not good to buy a game only to find out it doesn't work or to have to research a game you really want only to find out it doesn't work. Its just cutting out all of that.

I was thinking the game could just be hidden if its known to fail to start or not white listed, etc. But if the user uses both Windows and Linux and it would be used with Proton then it is indeed good to know. But yeah, there's already ways to find this out so its just duplication though it could be automated by Steam which would be nice - if uses can't start game, game is flagged as non-start and shows up as such.

**Comment by Zorrototo** [comment]
In case someone still doesn't know there is www.protondb.com to see the Proton rating. I also don't think it is relevant to add it directly to Steam, as for native games, I think it is up to the user to do his research and see if the user system is compatible or sufficient for the game to run.

**Comment by jcc10** [comment]
Even if this was not added, it would be nice to be able to mod the client to add it in.

I already had a large library before switching to Linux so I'm not looking for a way to view the stats via the store page, I personally just want a easy way to view the rating and date of last rating. (IE: If tested last week and it didn't work, why bother. If tested last year and it didn't work, might try. If going up, Might try. It suddenly stopped working? Is it going down?) Being able to see this information without jumping between the client and the browser would be nice.

**Comment by vlandemart** [comment]
I agree, we need some "Steam Play" status in store, and maybe easier way to navigate to troubleshooting of this title on Steam Play (right now it's much easier to open a browser and search a problem you are having).

Currently, I use protondb for all that, but I would REALLY like to see more information inside of steam, without the need to open external websites and etc.
Also, I haven't found any other ways to sort your library via "steam play statuses" but this: https://github.com/CorruptComputer/ProtonDB-Tags

**Comment by HonkingGoose** [comment]
I suspect this issue can be closed.

Steam now shows the Steam deck compatibility status in the store, and you can filter on Steam Deck status in your Library.

**Comment by jcc10** [comment]
I would like to add for future people, the decky plugin [For ProtonDB Badges](https://plugins.deckbrew.xyz/#:~:text=to%20do%20so.-,ProtonDB%20Badges,-Author%3A%20OMGDuke) solves the problem for not viewing ProtonDB status on the steam deck, though it would still be nice for steam to have a way to mod the info into the standard interface for 'linux-but-not-steamdeck' users.
